apps/dev-portal-web/pages/apps/b2b/[id].tsx

```
@@ -0,0 +1,136 @@
+import { Col, type MenuProps, Row, RowProps } from 'antd'
+import get from 'lodash/get'
+import Error from 'next/error'
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)